numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -1115,6 +1115,14 @@ _pyarray_correlate(PyArrayObject *ap1, PyArrayObject *ap2, int typenum,
 
     n1 = PyArray_DIMS(ap1)[0];
     n2 = PyArray_DIMS(ap2)[0];
+    if (n1 == 0) {
+      PyErr_SetString(PyExc_ValueError, "first array argument cannot be empty");
+      return NULL;
+    }
+    if (n2 == 0) {
+      PyErr_SetString(PyExc_ValueError, "second array argument cannot be empty");
+      return NULL;
+    }
```

Comment:
Nit: indent should be a multiple of 4

Reply:
Fixing now; should the commit message include the same "main line"? something like:
```
BUG: raise ValueError for empty arrays passed to _pyarray_correlate

fix indentation
```
or  is
```
fix indentation
```
OK?

Reply:
Sorry for the noise; found the `MAINT:` commit acronym

Reply:
Really you should just squash (`git rebase -i HEAD^^` ) or amend the first commit when the style you're fixing up was introduced in the same PR. Don't worry about it though, we can squash it with the merge button on github